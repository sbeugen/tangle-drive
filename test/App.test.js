import { shallow } from 'vue-test-utils'
import App from '../src/App'

describe('App.test.js', () => {
  let cmp
  
  beforeEach(() => {
    cmp = shallow(App)
  })

  it('has expected html structure', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})