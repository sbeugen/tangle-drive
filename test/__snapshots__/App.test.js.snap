// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.test.js has expected html structure 1`] = `
<div
  id="app"
>
  <router-view />
</div>
`;
